### ISSUE TITLE
v1.2.0 loads from NuGet as version `0.0.0.0` 

### ISSUE THREAD

**Issue by ebussa** [mgravell/fast-member#49]
This causes the Visual Studio compiler warning of `Found conflicts between different versions of "FastMember.Signed" that could not be resolved.` because ClosedXML expects >=1.1.0

**Comment by mgravell** [comment]
Well shoot. I'll take a look.

**Comment by tricus** [comment]
I'm having the same problem.  The auto-generated binding redirect:

```
        <assemblyIdentity name="FastMember.Signed" publicKeyToken="9e8f22703bef9a29" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-0.0.0.0" newVersion="0.0.0.0" />
```

**Comment by digitalvir** [comment]
I have ran into the same issue with the upgrade to 1.2.0

**Comment by dgubernatorov-softheme** [comment]
I've changed in web.config to this:

<dependentAssembly>
        <assemblyIdentity name="FastMember.Signed" publicKeyToken="9e8f22703bef9a29" culture="neutral" />
        <bindingRedirect oldVersion="1.0.0.9" newVersion="0.0.0.0" />
      </dependentAssembly> 

and it started to work.

**Comment by mgravell** [comment]
Sorry, this is fixed in 1.3.0
